lib/record_store/zone/config/ignore_pattern.rb

```diff
@@ -0,0 +1,41 @@
+module RecordStore
+  class Zone
+    class Config
+      class IgnorePattern
+        def initialize(orig_hash)
+          @orig_hash = orig_hash
+        end
+
+        def should_ignore?(record)
+          all_pairs.all?{|(key, value)| record.respond_to?(key) && value_matches?(value, record.send(key))}
+        end
+
+        def to_hash
+          orig_hash
+        end
+
+        private
+
+        attr_accessor :orig_hash
```

Comment:
Do you need this to be private? If you don't want to expose the `orig_hash` publicly, you should use the instance reference with `@orig_hash` in your code.

Reply:
My train of thought here:

- I created an `attr_accessor` for `orig_hash`
- I created a public accessor, `to_hash` for serializing
- Then I was like _orig_hash should be private_ and did this

I only need `to_hash`.  Imma dumb.

Reply:
Don't worry man!